packages/components/src/index.ts

```diff
@@ -761,6 +762,7 @@ export {
     getServerFilePreview,
     InferDomainResponse,
     IFieldChange,
+    DomainOnChange,
```

Comment:
It ends up getting used as the type for getGridColumns()'s onFieldsChange parameter, as well as in interface IDomainRowProps' onChange property. The only reason for the change was because the function signature was long and so making the getGridColumns() signature less readable, but I am also open to putting it back!

Reply:
Removed!